common/src/test/kotlin/org/wycliffeassociates/otter/common/TestUtils.kt

```diff
@@ -83,3 +99,150 @@ fun createTestWavFile(dir: File): File {
     return testFile
 }
 
+fun getDublinCore(resource: ResourceMetadata): DublinCore {
+    return dublincore {
+        conformsTo = "0.2"
+        identifier = resource.identifier
+        issued = LocalDate.now().toString()
+        modified = LocalDate.now().toString()
+        language = language {
+            identifier = resource.language.slug
+            direction = resource.language.direction
+            title = resource.language.name
+        }
+        creator = "Orature"
+        version = resource.version
+        rights = resource.license
+        format = MimeType.of(resource.format).norm
+        subject = resource.subject
+        type = resource.type.slug
+        title = resource.title
+    }
+}
+
+fun getResourceMetadata(langauge: Language): ResourceMetadata {
+    return ResourceMetadata(
+        conformsTo = "rc0.2",
+        creator = "Door43 World Missions Community",
+        description = "Description",
+        format = "text/usfm",
+        identifier = "ulb",
+        issued = LocalDate.now(),
+        language = langauge,
+        modified = LocalDate.now(),
+        publisher = "unfoldingWord",
+        subject = "Bible",
+        type = ContainerType.Bundle,
+        title = "Unlocked Literal Bible",
+        version = "1",
+        license = "",
+        path = File(".")
+    )
+}
+
+fun getEnglishLanguage(id: Int): Language {
+    return Language(
+        "en",
+        "English",
+        "English",
+        "ltr",
+        isGateway = true,
+        region = "Europe",
+        id = id
+    )
+}
+
+fun getSpanishLanguage(id: Int = 0): Language {
+    return Language(
+        "es",
+        "Spanish",
+        "Spanish",
+        "ltr",
+        isGateway = false,
+        region = "Europe",
+        id = id
+    )
+}
+
+fun getGenesisCollection(): Collection {
+    return Collection(
+        sort = 1,
+        slug = "gen",
+        labelKey = "project",
+        titleKey = "Genesis",
+        resourceContainer = null
+    )
+}
+
+fun buildWorkbook(
+    db: WorkbookRepository.IDatabaseAccessors,
+    source: Collection,
+    target: Collection
+) = WorkbookRepository(
+    mock(),
+    db
+).get(source, target)
+
+fun createWavFile(dir: File, name: String, data: ByteArray): File {
```

Comment:
Is creating physical WAV files necessary for testing?  IO actions make tests slow and expensive.  Could this be mocked with FileStreams or something similar?